hat/assets/js/apps/Iaso/domains/mappings/actions.js

```diff
@@ -126,6 +126,22 @@ export const applyPartialUpdate =
             });
     };
 
+export const applyUpdate = (mappingVersionId, payload) => dispatch => {
+    dispatch(fetchingMappingVersions(true));
+    return patchRequest(`/api/mappingversions/${mappingVersionId}/`, payload)
+        .then(() =>
+            dispatch(fetchMappingVersionDetail(mappingVersionId)),
+        )
+        .catch(err =>
+            dispatch(
+                enqueueSnackbar(errorSnackBar('fetchMappingsError', null, err)),
+            ),
+        )
+        .then(() => {
+            dispatch(fetchingMappingVersions(false));
+        });
+};
+
```

Comment:
you can use react-query for those now

Reply:
Sorry but I don't really have time to redo the whole screen. I prefer it keeps the same code the other actions then mixing the two.